src/Makefile.am

```diff
@@ -181,6 +181,7 @@ BITCOIN_CORE_H = \
   random.h \
   reverse_iterator.h \
   reverselock.h \
+  rpc/blockchain.h \
```

Comment:
Seems unrelated, no?

Reply:
Yes, but needed to test gitian build, which is broken without this header